charts/quantum-serverless/charts/gateway/templates/django-secret.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
maybe add something like {{ if .Values.application.django.secret_key }} to check wether we need this template

Reply:
what kind of scenario are you thinking of?

Reply:
if secret already configured in cluster

Reply:
or it is still will work?

Reply:
no, it will fail if the secret was previously created:

`Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: Secret "django" in namespace "default" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "qs"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "default"`

let me update that